.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2.1
+
+orbs:
+  gusto: gusto/gusto@volatile
```

Comment:
We probs want to explicitly pick a version here and then auto-PR updates with renovate. Orb is here: https://circleci.com/orbs/registry/orb/gusto/gusto